lib/ansible/plugins/action/copy.py

```diff
@@ -216,7 +216,7 @@ def _ensure_invocation(self, result):
             if self._play_context.no_log:
                 result['invocation'] = "CENSORED: no_log is set"
             else:
-                result["invocation"] = self._task.args.copy()
```

Comment:
I would keep this line so we preserve the existing behaviour without breaking people's playbooks. Best to add a comment saying we know this is wrong but need it to keep backwards compatibility.